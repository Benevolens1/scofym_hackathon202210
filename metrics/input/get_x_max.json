{
	"1": {
		"type": "get_x_max",
		"arguments": {"path":"data/media/x_max_1.txt","n": 1}
	},"2": {
		"type": "get_x_max",
		"arguments": {"path":"data/media/x_max_2.txt","n": 30}
	}
	,"3": {
		"type": "get_x_max",
		"arguments": {"path":"data/media/x_max_3.txt","n": 3}
	}
}