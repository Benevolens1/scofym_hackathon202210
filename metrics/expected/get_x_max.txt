1 ['a505']
2 ['a68960', 'a94835', 'a72451', 'a92058', 'a94149', 'a67214', 'a75204', 'a95811', 'a90634', 'a96135', 'a68864', 'a86385', 'a88544', 'a97487', 'a85946', 'a94000', 'a74172', 'a80705', 'a81462', 'a84988', 'a94604', 'a99897', 'a92667', 'a94048', 'a98046', 'a95503', 'a96828', 'a98571', 'a98390', 'a99897']
3 ['a7', 'a1', 'a3']
